contracts/registries/LicenseRegistry.sol

```diff
@@ -209,9 +209,6 @@ contract LicenseRegistry is ILicenseRegistry, AccessManagedUpgradeable, UUPSUpgr
         address licenseTemplate,
         uint256[] calldata licenseTermsIds
     ) external onlyLicensingModule {
-        if (parentIpIds.length == 0) {
```

Comment:
Don't we need this check? If `parentIpIds` is empty and the rest (two) of the if statements pass, the below code will get executed. If this behavior is desired, then the length check should be removed. Otherwise, it should remain.
```solidity
        $.licenseTemplates[childIpId] = licenseTemplate;
        // calculate the earliest expiration time of child IP with both parent IPs and license terms
        earliestExp = _calculateEarliestExpireTime(earliestExp, licenseTemplate, licenseTermsIds);
        // default value is 0 which means that the license never expires
        if (earliestExp != 0) _setExpirationTime(childIpId, earliestExp);
```

Reply:
It's protected upstream of this function